packages/transaction-controller/src/utils/transaction-type.ts

```diff
@@ -104,6 +113,12 @@ function parseStandardTokenTransactionData(
     // ignore and return undefined
   }
 
+  try {
+    return USDCInterface.parseTransaction({ data });
```

Comment:
I'd recommend a unit test usually for this extra logic, but it doesn't seem we have method specific test yet so happy to approve without.